receiver/api/src/main/java/org/phenotips/integration/lims247/script/RemoteDataScriptService.java

```diff
@@ -83,10 +83,19 @@ public void save()
             if (isTrusted()) {
                 try {
                     @SuppressWarnings("deprecation")
-                    XWikiDocument doc = new XWikiDocument();
-                    doc.fromXML(context.getRequest().getInputStream());
-                    context.getWiki().saveDocument(doc, doc.getComment(), doc.isMinorEdit(), context);
-                    this.logger.debug("Imported [{}] from remote", doc.getDocumentReference());
+                    XWikiDocument xmlDoc = new XWikiDocument();
+                    xmlDoc.fromXML(context.getRequest().getInputStream());
```

Comment:
Lines 86-87 make more sense to me down at 96.

Reply:
The document ID is needed for the existing document check on the next line.

Reply:
Ahh, missed that, sorry!